### PR TITLE
Fix sisyphus checking for empty list of disruptions

### PR DESCRIPTION
**Pull request by roydahan**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
Fixes #4333

File: test-cases/longevity/longevity-200GB-48h-network-monkey.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ nemesis_class_name: 'SisyphusMonkey'
 nemesis_include_filter: ['networking']
 nemesis_seed: '002'
 nemesis_interval: 10
+nemesis_multiply_factor: 15
```
Comment (fgelcer) [inline test-cases/longevity/longevity-200GB-48h-network-monkey.yaml:17]: for this job, i'm not even sure 15 will be enough...
if the default "6" ran for ~ 10 hours (in a 48 hours job), we will need it to be at least some 30, but i would raise it to some 40 just to be sure

Comment (roydahan) [reply test-cases/longevity/longevity-200GB-48h-network-monkey.yaml:17]: 15 is good enough.

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**